executors.go

```diff
@@ -20,10 +20,16 @@ func (c *Connection) Reload(model interface{}) error {
 	})
 }
 
+// TODO: consider merging the following two methods.
```

Comment:
can these two methods be merged to be based on `ExecWithCount()`? by just dropping the int return value?
It could be better to manage but the only issue is the elapsed time calculated by `timeFunc` could be slightly different :-)